### ISSUE TITLE
Latest code from ioquake3

### ISSUE THREAD

**Issue by twolife** [raspberrypi/quake3#2]
Hi,

i don't know what is the version you used but it's obviously not the latest ioq3 code.
i did try to apply your mods to the latest ioq3 svn code & it seems to work fine
(single player is ok, didn't test the rest of the game/menu)

in case you are interested :
http://twolife.be/debian/todo/ioquake3/patches/ioquake3-1.36+svn2304_rpi.patch

Regards

Sébastien

**Comment by Ruffio** [comment]
@twolife is this still an issue?

**Comment by NF3RN0** [comment]
@twolife How are you able to make this work with open arena?

**Comment by CrescendoFang** [comment]
I forked ioquake3's latest git and added the makefile from @twolife git.
https://github.com/CrescendoFang/ioq3
But I haven't tried it yet since I dunno the step to compile this.

There's no build.sh like in current RasPi Quake3 port.

Anyone able to shed a light how to compile this one?

**Comment by ellisgl** [comment]
Yeah - would be nice to have an update. A lot of things have changed, lot of improvements. Of course SDL might be an issue.
